app/services/food_service.rb

```diff
@@ -0,0 +1,12 @@
+class FoodService
+  def self.conn
+    Faraday.new('https://api.nal.usda.gov/fdc/v1/') do |request|
+      request.params['api_key'] = ENV['FOOD_API']
+    end
+  end
+
+  def self.searched_foods(food)
+    response = conn.get("foods/search?query=#{food}&ingredients=#{food}")
```

Comment:
You only need one of these queries -- we're specifically looking for ingredients that contain that food item.